### PR TITLE
Add manual test summary to test report (#124)

### PR DESCRIPTION
**Pull request by noursaidi**
Issue #124 
Implementation (inc Travis test) for adding summary of manual tests to the DAQ test report as per suggestion in #124. A new test module is added which reads manual test report data inputted into module_config.json and then produces DAQ results so that it is inserted into the test report.

### CODE REVIEW

**Comment by pisuke** [comment]
Thank you @noursaidi, it looks good to me and I'll test it tomorrow.
@grafnu this is a new feature to add manual report output, do you have any comments on the implementation?

**Comment by pisuke** [comment]
All tests passed and no comment received so merging to master.

File: docs/device_report.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Overall device result FAIL
 |skip|connection.port_duplex|Other|Other|No local IP has been set, check system config|
 |skip|connection.port_link|Other|Other|No local IP has been set, check system config|
 |skip|connection.port_speed|Other|Other|No local IP has been set, check system config|
+|pass|manual.test.travis|Security|Recommended|Manual test - for testing|
```
Comment (grafnu) [inline docs/device_report.md:73]: This should not explicitly reference "travis" -- does just "manual.test.sample" work?

File: subset/manual/readme.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Manual Tests
+
+
+## Manual Tests
+Some tests cannot be automated with DAQ although these may be required. To facilitate a single test report which incorporates all tests undertaken on a device, the `manual` test can be used to input the results into reports produced by DAQ.
```
Comment (grafnu) [inline subset/manual/readme.md:5]: break into multiple lines

File: subset/manual/readme.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Manual Tests
+
+
+## Manual Tests
+Some tests cannot be automated with DAQ although these may be required. To facilitate a single test report which incorporates all tests undertaken on a device, the `manual` test can be used to input the results into reports produced by DAQ. 
+
+## Configuration
+Manual tests including results are inserted into the device's `module_config.json` and marked by `"type": "manual"`.
```
Comment (grafnu) [inline subset/manual/readme.md:8]: multiple lines

File: subset/manual/readme.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    }
+}
+```
+
+
+## Example Report of Manual Tests
+In summary table
+|Result|Test|Category|Expectation|Notes|
+|---|---|---|---|---|
+|pass|manual.test.name|Security|Recommended|Manual test|
```
Comment (grafnu) [inline subset/manual/readme.md:29]: manual.test.name is good -- can use that instead of manual.test.travis

**Comment by grafnu** [comment]
I added some comments --
https://github.com/faucetsdn/daq/pull/481#pullrequestreview-434514330 -- we
have to be careful about submitting things without some eyes on them just
b/c there's so many emails going by that things can get lost... all fine as
long as the changes aren't significant and/or we're willing to go back and
clean them up after!  If you can take a look at my comments and make a few
minor changes, that'd be great, thanks!

**Pull request merged**